incubator/binding-openapi/src/main/java/io/aklivity/zilla/runtime/binding/openapi/internal/generator/model/ResponseByContentType.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2021-2023 Aklivity Inc
+ *
+ * Licensed under the Aklivity Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ *   https://www.aklivity.io/aklivity-community-license/
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package io.aklivity.zilla.runtime.binding.openapi.internal.generator.model;
+
+import java.util.LinkedHashMap;
+
+public class ResponseByContentType
+{
+    public LinkedHashMap<String, Header> headers;
+    public LinkedHashMap<String, Response> content;
```

Comment:
These types should be interface `Map<...>` instead of implementation `LinkedHashMap<...>`.